jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -512,60 +512,240 @@ def test_linear_solve(self, harness: primitive_harness.Harness):
 
     self.ConvertAndCompare(harness.dyn_fun, a, b, atol=tol, rtol=tol)
 
-  @primitive_harness.parameterized(primitive_harness.lax_unary_elementwise)
-  def test_unary_elementwise(self, harness: primitive_harness.Harness):
+  @primitive_harness.parameterized(primitive_harness.lax_abs)
+  def test_abs(self, harness: primitive_harness.Harness):
+    self.ConvertAndCompare(harness.dyn_fun, *harness.dyn_args_maker(self.rng()))
+
+  def _preimage_function_test_util(self, harness: primitive_harness.Harness,
+                                   atol=None, rtol=None):
+    inverse_fun = {lax.acosh_p: np.cosh,
+                   lax.asinh_p: np.sinh,
+                   lax.atanh_p: np.tanh,
+                   lax.acos_p: np.cos,
+                   lax.asin_p: np.sin,
+                   lax.atan_p: np.tan}[harness.params["prim"]]
+    custom_assert = None
+    operand, = harness.dyn_args_maker(self.rng())
+    if harness.params["dtype"] in [np.complex64, np.complex128]:
+      def custom_assert(result_jax, result_tf):
+        # There are technically infinitely many possible results, so we check
```

Comment:
I think that we ought to document that JAX and TF may return different answers for these ops.

Reply:
In that case, maybe we should do the same for `eig` and the likes, and every function that may return different but equally valid results. WDYT?

Reply:
Yes, that is a good idea. It would be good to document those ops for which we may get a different result. These should be exactly those that have custom asserts, no?

Reply:
I think this is true, yes.

Reply:
Should we add those limitations in this PR, or a future one?

Reply:
I just added it here! Sorry for the delay :)